### ISSUE TITLE
Pipenv ignores the `PIPENV_SPINNER` environment variable.

### ISSUE THREAD

**Issue by superruzafa** [pypa/pipenv#4045]
### Issue description

The [documentacion says](https://pipenv.kennethreitz.org/en/latest/advanced/#pipenv.environments.PIPENV_SPINNER) that the `PIPENV_SPINNER` environment variable can modify the animation shown while performing long time operations, but `pipenv` ignores that value.

### Expected result

~~~ bash
$ export PIPENV_SPINNER=arrow3
$ pipenv lock
▹▹▹▹▹ Locking... # <-- arrow3 spinner style
~~~

### Actual result

~~~ bash
$ export PIPENV_SPINNER=clock
$ pipenv lock
⠙ Locking... # <-- dots spinner style
~~~

### Steps to replicate

1. export the `PIPENV_SPINNER` variable to any [valid spinner value](https://github.com/sindresorhus/cli-spinners/blob/master/spinners.json).
2. run `pipenv`
3. observe that the dots (or the bouncingBar) spinner style is still present.

**Comment by allanlewis** [comment]
Presumably this is fixed now that #4046 is merged?

**Comment by oz123** [comment]
Closing as this is resolved.
